build.gradle

```diff
@@ -42,7 +42,7 @@ dependencies {
 }
 
 group = 'org.labkey.build'
-project.version = "6.2.0-SNAPSHOT"
+project.version = "6.2.0"
```

Comment:
The version should be chaged to `6.3.0-SNAPSHOT` before merging.
I'm not sure that this should have ever been set to a non-snapshot version in the branch. I usually merge it with the next snapshot version then @labkey-susanh publishes the release version (not sure exactly what that process is).

Reply:
Yes, that's the process. Once this `gradlePlugins` branch is merged (using the next snapshot version), I'll publish the 6.2.0 version and you can update your server branch to use that version and merge it.

Reply:
OK, version here is now 6.3.0-SNAPSHOT

Reply:
No, this branch is fine. This changing of the `project.version` property to `6.3.0-SNAPSHOT` is just in preparation for merging to develop. When publishing the plugin, I'll make a separate branch that populates this with the release version.

Reply:
Thank you!